relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -201,6 +204,14 @@ pub fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
 
     if MOBILE_SDKS.contains(&event.sdk_name()) {
         tags.insert(SpanTagKey::Mobile, "true".to_owned());
+
+        if event.context::<AppContext>().is_some() {
```

Comment:
Might be worth adding a comment here why we limit this to apps.

> check if contexts.app exists - this tells us if the span originated from an app (as opposed to mobile browser - since we are currently focused on app use cases only)